dts/bindings/psi5/psi5-controller.yaml

```
@@ -0,0 +1,141 @@
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: PSI5 (Peripheral Sensor Interface) Controller
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    Each child node defines the configuration of a PSI5 channel
+
+  properties:
+    reg:
+      type: int
+      required: true
+      description: Channel identifier.
+
+    async-mode:
+      type: boolean
+      description: |
+        Determines the channel operation mode. When set to true, the channel operates in
+        asynchronous mode with only the receive function active. When set to false,
+        the channel operates in synchronous mode with both transmit and receive functions active.
+
+    period-sync-pulse-us:
+      type: int
+      description: |
+        Specifies the period of the internally generated synchronization pulses, measured in
+        microseconds (us). This value determines the length of each synchronization pulse used
+        in the system.
+
+    pulse-width-0-us:
+      type: int
+      description: |
+        Specifies the duration of the pulse width for a data bit value '0', measured in
+        microseconds (us).
+
+    pulse-width-1-us:
+      type: int
+      description: |
+        Specifies the duration of the pulse width for a data bit value '1', measured in
+        microseconds (us).
+
+    decoder-start-offset-us:
+      type: int
+      description: |
+        Specifies the duration for which the Manchester decoder remains inactive after the
+        falling edge of a synchronization pulse, measured in microseconds (us). This value
+        determines the delay before the decoder starts processing incoming signals again.
+
+    tx-frame:
+      type: string
+      enum:
+        - short-31-1s
```

For string, you should not use implicit.
```suggestion
        - "short-31-1s"
```
(And other.)